<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HDF5LineImport_Widget</class>
 <widget class="QWidget" name="HDF5LineImport_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDF5 Line Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>HDF5 Loading</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="load_single_hdf5_line_button">
     <property name="text">
      <string>Load Single HDF5 Line</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="multiFileLayout">
     <item>
      <widget class="QPushButton" name="load_multi_hdf5_line_button">
       <property name="text">
        <string>Load Multiple HDF5 Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="multi_filename_pattern_text">
       <property name="placeholderText">
        <string>*.h5</string>
       </property>
       <property name="text">
        <string>*.h5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pattern">
       <property name="text">
        <string>Filename Pattern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
